### ISSUE TITLE
Eliminate/Restrict Jquery use for cornerstone-tools

### ISSUE THREAD

**Issue by louisgv** [concept-to-clinic/concept-to-clinic#215]
## Expected Behavior

Related to #197 per @vessemer request.

The front-end should not be depending on JQuery as a dependence, unless there is popular demand from contributor. Per #197, it is being imported due to another dependency for advanced medical image processing called `cornerstone-tool`.

It is desirable if we can devise a minimal version of JQuery that has all the required function that CornerStone-tool need, or simply find a way to remove JQuery.

## Current Behavior

The front-end application starting after #197 will be relying on JQuery as a dependency.

## Possible Solution

+ Searching for `external.$` in `cornerstone-tool` [repository](https://github.com/chafey/cornerstoneTools), then log down all method the library is relying on.
+ Extract those method from jQuery and export them into an internal library, with which we expose it to cornerstoneTools
+ Potentially make PR upstream to cornerstone-tool

## Context (Environment)

+ JQuery is a sizable library, however we're already using vue.js and since most of the feature we need from JQuery can easily be implemented with vue, it is necessary to not be depended on jQuery.

## Checklist before submitting
- [ + ] I have confirmed this using the officially supported Docker Compose setup using the `local.yml` configuration and ensured that I built the containers again and they reflect the most recent version of the project at the `HEAD` commit on the `master` branch 
- [ + ] I have searched through the other currently open issues and am confident this is not a duplicate of an existing bug
- [ + ] I provided a **minimal code snippet** or list of steps that reproduces the bug.
- [ ] I provided **screenshots** where appropriate
- [ + ] I filled out all the relevant sections of this template

**Comment by truefalse10** [comment]
is anyone working on this right now? otherwise i would start

**Comment by Serhiy-Shekhovtsov** [comment]
@truefalse10 btw, looks like they have a branch for this: https://github.com/chafey/cornerstone/tree/zachasme-es6
More details [here](https://github.com/chafey/cornerstone/issues/180) and [here](https://github.com/chafey/cornerstone/issues/92)

**Comment by dannyrb** [comment]
It's worth noting that cornerstone is on track to drop its dependency on jQuery. Work is actively being done. It might be worth waiting for the official change, and then update to latest.

**Comment by swederik** [comment]
We just released Cornerstone Tools 2.0.0 which no longer depends on jQuery. Let us know if you run into any bugs. Thanks!

**Comment by lamby** [comment]
https://github.com/concept-to-clinic/concept-to-clinic/commit/ca6f6549f92eca12a7d8c37daab7284d76379bba :)

**Comment by dannyrb** [comment]
@lamby, if you have other Cornerstone related requests, please feel free to cross-post issues. Always happy to hear about others using Cornerstone, and we want to do what we can to make it more useful for integrators

**Comment by Serhiy-Shekhovtsov** [comment]
This can be closed. Implemented in #269.
